pkg/openshift-sdn/informers.go

```diff
@@ -42,7 +43,10 @@ func (sdn *OpenShiftSDN) buildInformers() error {
 	}
 
 	kubeInformers := kinformers.NewSharedInformerFactory(kubeClient, sdn.ProxyConfig.IPTables.SyncPeriod.Duration)
-	networkInformers := networkinformers.NewSharedInformerFactory(networkClient, defaultInformerResyncPeriod)
+	networkInformers := networkinformers.NewSharedInformerFactoryWithOptions(networkClient, defaultInformerResyncPeriod,
+		networkinformers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.LabelSelector = "!service.kubernetes.io/service-proxy-name"
+		}))
```

Comment:
So as you already noticed, you wanted to change `kubeInformers` not `networkInformers`, but you can't do it like this anyway; these informers get used by *all* of the sdn, so if you filter out some services here, then, the sdn Service handling code won't see them either. That's why I was saying to filter them out from `pkg/network/proxy/` instead.

Reply:
Well, I was thinking of doing two separate PRs, just so we have a 1-1 for https://github.com/kubernetes/kubernetes/pull/71355/files and https://github.com/kubernetes/kubernetes/pull/81612/files .
The filtering for endpoints would come in  other PR.